packages/network/src/connection/webrtc/WebRtcConnection.ts

```diff
@@ -7,11 +7,18 @@ import { MessageQueue, QueueItem } from '../MessageQueue'
 import { NameDirectory } from '../../NameDirectory'
 import crypto from 'crypto'
 
+export interface IceServer {
+    url: string
+    port: number
+    username?: string
+    credential?: string
```

Comment:
Do we want to support multiple credential types (by adding an optional `credentialType` field, https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer)?

Maybe it is best to use username+password authentication as we currently do. But then we could consider naming `credential` to `password`.

Reply:
Yeah make sense